swagger/api_v3/query_service.swagger.json

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
note that we already host zipkin swagger that is in yaml format https://github.com/jaegertracing/jaeger-idl/tree/master/swagger. 

the protoc cannot generate yaml, we could translate the file using `yq`, I do not fancy it very much.